addons/web_dashboard/static/src/less/dashboard.less

```diff
@@ -0,0 +1,122 @@
+.o_web_dashboard {
+    .o_dashboard_card() {
+        box-shadow: 0 1px 2px 0 rgba(0,0,0,.14);
+        background: @bg-dashboard-col;
+        border-radius: 2px;
+    }
+
+    @bg-dashboard-col: #FFF;
+    background: #F7F7F7;
+    height: 100%;
+    padding: 0px 5px;
+    overflow: auto;
+
+    .o_web_dashboard_col {
+        padding: 10px 8px;
+        > div:first-child {
+            .o_dashboard_card();
+            padding: 25px 18px 18px 18px;
+                .o_web_dashboard_header {
+                    font-size: 22px;
+                    margin-bottom: 8px;
+                }
+        }
+        .o_web_dashboard_pills {
+            margin-top: 10px;
+            i.fa {
+                vertical-align: middle;
+                font-size: 1.5em;
+            }
+            a {
```

Comment:
don't use css selectors with html tags if you do not help the browser to tie break quickly by adding a class or using the ">" selector. you have this probleme multiple times in your less file here